src/Fixer/ClassNotation/OrderedClassElementsFixer.php

```diff
@@ -436,6 +470,21 @@ private function sortElements(array $elements)
         return $elements;
     }
 
+    private function sortGroupElements(array $a, array $b)
+    {
+        $selectedSortAlgorithm = $this->configuration['sortAlgorithm'];
+
+        if (!in_array($selectedSortAlgorithm, $this->supportedSortAlgorithms, true)) {
```

Comment:
please check this one time during configuration, after that it is not needed to re-check the config

Reply:
is it already verified by `ConfigurationResolver`

Reply:
ok